src/services/CertificateGenerationService.ts

```diff
@@ -51,14 +51,9 @@ import { S3BucketService } from "./S3BucketService";
  */
 @Service()
 class CertificateGenerationService {
-  private readonly s3Client: S3BucketService;
-  private readonly config: Configuration;
-  private readonly lambdaClient: LambdaService;
-
-  constructor(s3Client: S3BucketService, lambdaClient: LambdaService) {
-    this.s3Client = s3Client;
-    this.config = Configuration.getInstance();
-    this.lambdaClient = lambdaClient;
+  private readonly config: Configuration = Configuration.getInstance();
+
+  constructor(private s3Client: S3BucketService, private lambdaClient: LambdaService) {
```

Comment:
I know the implementations for dependency injection are slightly different, but from what we did on the central docs repo, is the @Inject() decorator not also needed here when using `typedi`?

Reply:
Correct, it is not. We can actually make all the injections private too to simplify (this was in a subsequent commit).